src/modules/rewards/components/RewardsTabsContainer.tsx

```diff
@@ -13,17 +13,20 @@ import { RewardsActivitiesSection } from './RewardsActivitiesSection';
 //Types
 import { RewardsTabs as RewardsTabsType } from '../Rewards.types';
 import { UserStoreType } from 'types';
+import { UserRewardsDetailResponse } from 'queries';
 
 export type RewardsTabsContainerProps = {
   activeTab: RewardsTabsType;
   handleSetActiveTab: (tab: RewardsTabsType) => void;
   setShowConnectModal: Dispatch<SetStateAction<boolean>>;
+  userDetails: UserRewardsDetailResponse | undefined;
```

Comment:
```suggestion

```

Remove this. @abhishek-01k please improve this habit of yours to not removing unused codebase.